src/main/java/Duke.java

```diff
@@ -1,10 +1,264 @@
+import src.main.java.*;
```

Comment:
Perhaps you can import individually and avoid wildcards, to be more explicit about what will be used.